C/CeresSolver/build_tarballs.jl

```diff
@@ -26,33 +21,27 @@ else
 fi
 
 CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix}
+              -DCMAKE_CXX_FLAGS="-std=c++17"
```

Comment:
You should rather use [`CMAKE_CXX_STANDARD`](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD).  Also, if this is really needed, this option should be set by upstream CMake file, it doesn't make sense that users need to set this option manually.